src/main/resources/db/migration/V69__aldersvilkår_målgruppe.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE vilkar_periode
+SET fakta_og_vurdering = jsonb_set(fakta_og_vurdering, '{vurderinger, aldersvilkår}', '{
+"svar": "GAMMEL_MANGLER_DATA",
+"resultat": "OPPFYLT",
+"vurderingFaktaEtterlevelse": null
+}');
```

Comment:
Her tror jeg alle vilkårperiodene vil få aldersvilkåret. Det er vel kun målgruppe som skal ha det? Aktivitet skal ikke ha aldersvilkår?

Reply:
God catch! Har testet lokalt nå, og kan bekrefte at Aktiviteter også populeres med Aldersvilkår slik det er nå. Noen tanker rundt hvordan det bør skrives om for å bare treffe Målgrupper? @sberbom

Reply:
Tror dette skal funke, skal teste lokalt: c9f1e2fc377329cdc4be13bc6a46530318cf08f5

Reply:
Har nå testet lokalt for:
`OMSTILLINGSSTØNAD_TILSYN_BARN`,`OVERGANGSSTØNAD_TILSYN_BARN`, `AAP_TILSYN_BARN`, `TILTAK_TILSYN_BARN`
Da gikk det som forventet at bare `AAP_TILSYN_BARN` og `OMSTILLINGSSTØNAD_TILSYN_BARN` ble populert med Aldersvilkår

Reply:
Kjørte ny lokal migrering for:
`TILTAK_LÆREMIDLER`, `AAP_LÆREMIDLER`, `OVERGANGSSTØNAD_LÆREMIDLER` og `OMSTILLINGSSTØNAD_LÆREMIDLER`

Som forventet var det bare: `OMSTILLINGSSTØNAD_LÆREMIDLER` og `AAP_LÆREMIDLER` som ble populert med Aldersvilkår